packages/ra-ui-materialui/src/button/BulkDeleteWithConfirmButton.tsx

```diff
@@ -39,8 +39,8 @@ const BulkDeleteWithConfirmButton: FC<
     const {
         basePath,
         classes: classesOverride,
-        confirmTitle,
-        confirmContent,
+        confirmTitle = 'ra.message.bulk_delete_title',
```

Comment:
Why that change? Also, why do it for confirmTitle and not for label?

Reply:
I was using type-script with strictMode = true. I'll change it.